src/rand.rs

```diff
@@ -323,22 +328,13 @@ mod tests {
             }
 
             // Make sure we didn't forget to finish filling in the rest of the
-            // buffer after we filled in the first chunk, especially in the
+            // buffer after we filled in the first chunk.
             // case in the `SysRandOrDevURandom::Undecided` case. As above, we
             // only do this when there are at least 96 bytes after the first
             // chunk to avoid false positives.
-            if *len > 96 && *len - 96 > max_chunk_len() {
-                assert!(buf[max_chunk_len()..].iter().any(|x| *x != 0));
+            if *len > 96 {
+                assert!(buf[buf.len() - 96 ..].iter().any(|x| *x != 0));
```

Comment:
I think we should just get rid of this. It really is about max_chunk_len() and it doesn't make sense now.